src/live_restore/live_restore_private.h

```diff
@@ -87,6 +87,7 @@ struct __wt_live_restore_fs {
     WT_LIVE_RESTORE_FS_LAYER destination;
     WT_LIVE_RESTORE_FS_LAYER source;
 
+    bool debug_fill_holes_on_close;
```

Comment:
Should maybe a uint8_t flags field for future expansion?

Reply:
Changed in [032235b](https://github.com/wiredtiger/wiredtiger/pull/11332/commits/032235bdbfca37396e0b27efdba71fb13308b7b3)